h2o-persist-gcs/testMultiNode.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
@michalkurka we should get rid of this file . WDYT? It is copied 5+ times?

Reply:
@mmalohlava We should but refactoring shouldn't be done in the feature PR.

@Pscheidl can you please make a technical jira to remove this file? The easiest is to generate it from a template? I would start with the parsers only.

Reply:
Will do.